### PR TITLE
Add links to CLRIE released msi/msm files

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**
v.1.0.31 release

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# CLR Instrumentation Engine Releases
+
+The below links provide the latest released packages for CLRIE. Some Azure platforms already support CLRIE as a first part extension. (see [here](scenarios/azure.md) for details).
```
Comment (jander-msft) [inline docs/releases.md:3]: first party extension*

File: docs/releases.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+## Current Release - 1.0.31
+
+|Platform|Links|
```
Comment (jander-msft) [inline docs/releases.md:7]: |Platform|Installer|Merge Module| ?

Then you can just change the links to [Download]

Comment (WilliamXieMSFT) [reply docs/releases.md:7]: I was trying to figure out a way to format it properly. I like your suggestion

**Pull request merged**